frontend/wallet/src/render/components/SideBar.re

```diff
@@ -40,53 +40,56 @@ let make = () => {
     React.useContext(AddressBookProvider.context);
 
   <div className=Styles.sidebar>
-    <WalletQuery>
-      {response =>
-         switch (response.result) {
-         | Loading => React.string("...")
-         | Error(err) => React.string(err##message)
-         | Data(data) =>
-           <WalletList
-             wallets={Array.map(~f=Wallet.ofGraphqlExn, data##ownedWallets)}
-           />
-         }}
-    </WalletQuery>
-    <div className=Styles.footer>
-      <Link onClick={_ => setModalState(_ => Some("My Wallet"))}>
-        {React.string("+ Add wallet")}
-      </Link>
-    </div>
-    <AddWalletMutation>
-      {(mutation, _) =>
-         switch (modalState) {
-         | None => React.null
-         | Some(newWalletName) =>
-           <AddWalletModal
-             walletName=newWalletName
-             setModalState
-             onSubmit={name => {
-               let performMutation =
-                 Task.liftPromise(() =>
-                   mutation(~refetchQueries=[|"getWallets"|], ())
+    // TODO: Remove fetchPolicy="no-cache" after merge of
+    // https://github.com/apollographql/reason-apollo/pull/196
```

Comment:
epic!

Reply:
Haha intense writeup for a 1-line fix too.